SofaKernel/modules/SofaCore/src/sofa/core/objectmodel/Data.h

```diff
@@ -113,6 +113,7 @@ class Data : public BaseData
     explicit Data(const InitData& init) : BaseData(init)
     {
         m_value = ValueType(init.value);
+        m_hasDefaultValue = true;
```

Comment:
So I guess this constructor is used when there is a default value, and the other constructors don't initialize a default value. Am I right?